packages/client/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "module": "es2020",
+    "target": "es2020",
```

Comment:
Redundant with the configuration of extended config.

Reply:
Removed.